--- kmyirc-0.2.9/configure.in.in.orig	Thu Feb 13 16:45:46 2003
+++ kmyirc-0.2.9/configure.in.in	Tue May 13 10:34:06 2003
@@ -2,8 +2,6 @@
 
 AM_INIT_AUTOMAKE(kmyirc,0.2.9)
 
-CXXFLAGS="$NOOPT_CXXFLAGS" dnl __kdevelop[noopt]__
-CFLAGS="$NOOPT_CFLAGS" dnl __kdevelop[noopt]__
 dnl CXXFLAGS="$CXXFLAGS $USE_EXCEPTIONS" dnl __kdevelop[exc]__
 
 dnl KDE_NEED_FLEX dnl __kdevelop__
--- kmyirc-0.2.9/configure.in.orig	Tue Apr 22 05:50:41 2003
+++ kmyirc-0.2.9/configure.in	Tue May 13 10:34:21 2003
@@ -59,8 +59,6 @@
 
 dnl PACKAGE set before
 
-CXXFLAGS="$NOOPT_CXXFLAGS" dnl __kdevelop[noopt]__
-CFLAGS="$NOOPT_CFLAGS" dnl __kdevelop[noopt]__
 dnl CXXFLAGS="$CXXFLAGS $USE_EXCEPTIONS" dnl __kdevelop[exc]__
 
 dnl KDE_NEED_FLEX dnl __kdevelop__
